bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/ChannelTypeUtils.java

```diff
@@ -96,6 +96,10 @@ public static State toOnOffType(@Nullable Integer value) {
         return value != null ? (value == 1 ? OnOffType.ON : OnOffType.OFF) : UnDefType.NULL;
     }
 
+    public static State toOnOffType(@Nullable Boolean value) {
+        return value != null ? (value ? OnOffType.ON : OnOffType.OFF) : UnDefType.NULL;
```

Comment:
UnDefType.NULL should never be set. Please rather use UnDefType.UNDEF.

Can you fix the other methods in the class please ? I see the same error at line 96 for example.

Reply:
Ok, I have just realized the same behavior like the other methods. I can change that when you are sure. I hope this pull-request is not a never ending story... ;-)

Reply:
done (but I have only changed it for toOnOffType methods because there are too many usages and I can't ensure that nothing would break. That could be a good separate refactoring ;-))